Applications/Help/ManualModel.cpp

```diff
@@ -6,7 +6,8 @@
 
 static ManualSectionNode s_sections[] = {
     { "1", "Command-line programs" },
-    { "2", "System calls" }
+    { "2", "System calls" },
+    { "8", "System administration commands and daemons" }
```

Comment:
This is a long name.. does it look OK in the Help app?

Reply:
It's cut off but you can expand the view of course. That's the "standard" name kind of. Is there a different one you'd prefer?